src/Services/UserLoginService.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Blumilk\Meetup\Core\Services;
+
+use Blumilk\Meetup\Core\Models\User;
+
+use Illuminate\Auth\AuthenticationException;
+use Illuminate\Http\Response;
+use Illuminate\Support\Facades\Auth;
+use Illuminate\Support\Facades\Hash;
+
+class UserLoginService
+{
+    /**
+     * @throws AuthenticationException
+     */
+    public function loginUser($data): Response
+    {
+        $user = User::where("email", $data["email"])->first();
+        if (!$user || !Hash::check($data["password"], $user["password"])) {
```

Comment:
```suggestion
        $user = User::where("email", $email)->first();
        if (!$user || !Hash::check($data["password"], $password)) {
```
Its looks better than getting something from array.